src/components/Dashboard.js

```diff
@@ -1,18 +1,23 @@
 import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
 import DashNav from './DashNav'
 import LocalNav from './LocalNav'
 import DashboardBody from './DashboardBody'
 
 class Dashboard extends Component {
   render() {
+
     return (
       <div className="dashboard-home">
         <DashNav/>
         <LocalNav/>
-        <DashboardBody/>
+        <DashboardBody widgets={this.props.widgets} />
       </div>
     )
   }
 }
 
-export default Dashboard
+export default connect( state => {
+  return { widgets: state.widgets }
+})( Dashboard )
```

Comment:
This could be written
```
export default connect( state => ({widgets: state.widgets}) )( Dashboard )
```
or
```
const mapStateToProps = state => ({widgets: state.widgets})
export default connect( mapStateToProps )( Dashboard )
```